### PR TITLE
DTGB-547: Added icons overview.

### PR DESCRIPTION
**Pull request by zero2one**
Added the icons overview to the icon atom page.

## Description

Added overview of all icons as supported by the Ghent icon font.

## Motivation and Context

Make it easier for theme maintainers and users to know what the available icons are.

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [x] I have read the **CONTRIBUTING** document.
- [x] I have updated the documentation accordingly.
- [x] I have updated the style guide CHANGELOG accordingly.
- [ ] I have updated the gent_base CHANGELOG accordingly.
- [x] I have ran gulp axe on the compiled files and found no errors.

### CODE REVIEW

**Review by delrueba — CHANGES_REQUESTED** [review]
We zetten alle config files systematisch om naar config.js
Dan kan je variants aanmaken en de collate functie gebruiken om ze allemaal in één preview te stoppen in een description list.

Zie buttons en links.

**Comment by delrueba** [comment]
Tof! Zal de gebruiksvriendelijkheid ten goede komen.

Zou je na de refactor gulp axe willen laten lopen?
Ik heb alle fouten vorige week opgelost, dus je zou geen mogen hebben.

@meirege ging eens de pr description aanpassen daarvoor ;).

**Review by meirege — APPROVED** [review]

**Comment by meirege** [comment]
@delrueba @zero2one What's the status of this PR?

**Comment by zero2one** [comment]
* I replaced the .json config file by a .js config file.
* I replaced the list of icons by a list of variants.

File: styleguide/components/21-atoms/icon/icon.config.js [hunk 1]
```diff
@@ -0,0 +1,106 @@
...
+}
+
+module.exports = {
+  title: 'Icon',
+  status: 'ready',
+  collated: true,
+  collator: function (markup, item) {
+    return `<!-- Start: @${item.handle} -->\n<p>${markup} .icon-${item.name}</p>\n<!-- End: @${item.handle} -->\n`;
+  },
+  preview: '@preview',
```
Comment (delrueba) [inline styleguide/components/21-atoms/icon/icon.config.js]: Is there a reason you don't use the dl-preview and dl collator function?

**Review by delrueba — APPROVED** [review]

**Pull request merged**